zgw_consumers/models/services.py

```diff
@@ -113,6 +126,22 @@ def clean(self):
                 {"header_key": _("If header_value is set, header_key must also be set")}
             )
 
+    @property
+    def get_health_check_indication(self) -> bool:
+        from zgw_consumers.client import build_client
+
+        try:
+            client = build_client(self)
+            if (
+                client.get(self.api_health_check_endpoint or self.api_root).status_code
+                == 200
+            ):
+                return True
+        except (ConnectionError, RequestException) as e:
+            logger.exception(self, exc_info=e)
```

Comment:
`logger.exception(self)` doesn't make sense, you must always provide a log message to logger calls:

```py
logger.info("Encountered an error while performing the health check to service %s", self, exc_info=e)
```

I would also seriously lower the log level, this is non an uncaught exception that is crashing the application, this is us deliberately trying to trigger errors to prevent the former at runtime.